amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -9,7 +9,7 @@ use specs::{Component, DenseVecStorage, FlaggedStorage};
 /// Local position, rotation, and scale (from parent if it exists).
 ///
 /// Used for rendering position and orientation.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
That's nice to know!
I'll have to make sure that this only happens for the network though, because I don't want people to save their game with data that is inaccurate.